### PR TITLE
New dlg for islands

### PR DESCRIPTION
**Pull request by MisterSimple**

### CODE REVIEW

File: Monika After Story/game/script-islands-event.rpy [hunk 1]
```diff
@@ -240,19 +240,19 @@ label mas_monika_day1:
...
         m "This weather would be good for a little book reading under the Cherry Blossom tree right, [player]?"
-        if mas_current_weather == mas_weather_def:
-            m "Lying under the shade while reading my favorite book."
-            m "Along with a snack and your favorite drink on the side."
         if mas_is_snowing or mas_isWinter():
             m "Huddled together, so as to stave off the cold."
             m "With some nice hot drinks to help keep us warm."
+        else:
+            m "Lying under the shade while reading my favorite book."
+            m "Along with a snack and your favorite drink on the side."
```
Comment (jmwall24) [inline Monika After Story/game/script-islands-event.rpy]:
```suggestion
            m "...Along with a snack and your favorite drink on the side."
```

File: Monika After Story/game/script-islands-event.rpy [hunk 2]
```diff
@@ -385,17 +385,40 @@ label mas_island_bookshelf:
...
     return
 
 label mas_island_bookshelf2:
-    m "Reading outdoors is a nice change of pace, you know?"
-    m "I'd take a cool breeze over a stuffy library any day."
-    m "Maybe I should add a table underneath the Cherry Blossom tree."
-    m "It'd be nice to enjoy a cup of coffee with some snacks to go alongside my book reading."
-    m "That'd be wonderful~"
+    if mas_current_weather == mas_weather_overcast or mas_is_raining:
+        m "Reading indoors with rain just outside our window is pretty relaxing."
```
Comment (jmwall24) [inline Monika After Story/game/script-islands-event.rpy]:
```suggestion
        m "Reading indoors with rain just outside the window is pretty relaxing."
```

File: Monika After Story/game/script-islands-event.rpy [hunk 3]
```diff
@@ -385,17 +385,40 @@ label mas_island_bookshelf:
...
+        m "If only I hadn't left the books outside..."
+        m "I should probably bring some here when I get the chance."
```
Comment (jmwall24) [inline Monika After Story/game/script-islands-event.rpy]:
```suggestion
        m "I should probably bring some in here when I get the chance."
```

**Comment by multimokia** [comment]
Getting an extra pair of eyes due to a bit of code changes. Just want to make sure things flow correctly

**Pull request merged**